tests/test_parallelization.py

```diff
@@ -74,3 +74,35 @@ def test_parallel_subcommand_relative_ordering() -> None:
     assert thrd_sc["angles"] == [[0.5, 0.0], "pi"]
     assert frth_sc["qop"] == "RZ"
     assert frth_sc["angles"] == [[3.5], "pi"]
+
+
+def test_106_1qubit() -> None:
```

Comment:
Could you rename this test to describe its intent such as "no_parallel_1qubit" or similar?

We do not need to reference "issue106", "106", etc in the code. It will be available through git and github history/metadata.